Lib/bdb.py

```diff
@@ -515,10 +505,10 @@ def get_all_breaks(self):
     # to get a data structure representing a stack trace.
 
     def get_stack(self, f, t):
-        """Get a list of frames in the stack trace.
+        """Return a list of (frame, lineno) in a stack trace and a size.
 
-        Get a list of records for a frame and all higher (calling) and
-        lower frames, and the size of the higher part.
+        List starts with original calling frame, if there is one.
```

Comment:
Got it.  Thanks.  I think I need to figure out a call to get_stack to really understand it.